kitsune/questions/tests/test_templates.py

```diff
@@ -1411,96 +1410,6 @@ def test_localized_creation(self):
         question = Question.objects.filter(title='A test question')[0]
         eq_(question.locale, 'pt-BR')
 
-    def test_full_workflow_inactive(self):
```

Comment:
Removed this because an inactive user cannot enter the AAQ flow

Reply:
But what if a user with an active session is marked as inactive? In that case we will have an authenticated user that is not active

Reply:
The `aaq` flow now uses the `login_required` decorator. By default, this checks if the user is authenticated and `is_active`:
```
def login_required(func, login_url=None, redirect=REDIRECT_FIELD_NAME, only_active=True):
    if only_active:
        def redirect_func(user):
            return not (user.is_authenticated() and user.is_active)
    ...
```

With that in mind, it seems that a user that was authenticated but was not active would not be able to access the AAQ flow at all, and therefore, we would not need to perform `is_active` checks in the view itself. I added a test `test_inactive_user` to confirm that a user that becomes inactive, they cannot see the AAQ flow (`test_inactive_user`).

Let me know if I misunderstood something about this flow.